app/views/v2/common_cases_view.scala.html

```diff
@@ -18,78 +18,23 @@
 @import config.AppConfig
 @import views.html.includes.main
 @import views.html.components.heading
-@import views.html.components.common_atar
-@import views.html.components.common_liabilities
+@import views.html.components.my_cases_tab
 @import views.html.components.primary_navigation
-@import views.html.components.common_correspondence
-@import views.html.components.common_miscellaneous
+@import models.viewmodels.{ApplicationTabViewModel, MyCasesTab}
+
 
 @this(my_cases_secondary_nav: views.html.components.my_cases_secondary_navigation)
 
-@(title: String)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
+@(myCases : ApplicationTabViewModel)(implicit request: AuthenticatedRequest[_], messages: Messages, appConfig: AppConfig)
```

Comment:
Should this be called my_cases_view now?

Reply:
👍